app/main/views/audits.py

```diff
@@ -180,6 +212,48 @@ def acknowledge_audit(audit_id):
     return jsonify(auditEvents=audit_event.serialize()), 200
 
 
+@main.route('/audit-events/<int:audit_id>/acknowledge-including-previous', methods=['POST'])
```

Comment:
The implementation is generic and will work for all audit event types though, so putting it under `/services` seems odd to me. (In that you could inadvertently post to `/services...` and acknowledge a bunch of audit events completely unrelated to services.

I think the frontends should be responsible for when they call this and the API as clear as possible what it does.  Maybe we could have the route return the JSON-equivalent of a NotImplementedError if the type of the audit isn't one we're expecting (i.e. anything other than a service update event)? 

It does feel like this lives better under `/audit-events` than `/services`, but @allait I see your point about the verb referencing a single audit ID.  How about re-jigging the order to something like `/audit-events/acknowledge-including-previous/<audit_event_id>` (similar to what we have for, e.g. `/draft-services/copy-from/<service_id>`)?  Does that make it feel less like it implies operating on a single event?

Or we could go for something like a `POST /audit-events/acknowledge-all-for-object-until` with the `{"latest_audit_id": ...}` payload?